encoder/vaapiencoder_h264.cpp

```diff
@@ -1661,11 +1661,12 @@ bool VaapiEncoderH264::addPackedPrefixNalUnit(const PicturePtr& picture) const
 
     bit_writer_write_trailing_bits(&bs);
 
+    uint32_t codedBits = bs.getCodedBitsCount();
```

Comment:
@xuguangxin , jpeg has not call bit writer, such as getCodedBitsCount. For HEVC encoder,     bit_writer_write_trailing_bits(&bs) will pad some zeros, so it is not a issue.